jvm/local.properties

```diff
@@ -0,0 +1,8 @@
+## This file must *NOT* be checked into Version Control Systems,
```

Comment:
Can you remove this file?

Reply:
Added to gitignore because this gets generated on each project open usually